apdFlywayConfig.conf

```diff
@@ -1,4 +1,4 @@
 flyway.user=${SPRING_DATASOURCE_USERNAME}
 flyway.password=${SPRING_DATASOURCE_PASSWORD}
 flyway.schemas=${GPD_SCHEMA_NAME}
-flyway.url=${SPRING_DATASOURCE_URL}
+flyway.url=${FLYWAY_DATASOURCE_URL}
```

Comment:
`export SPRING_DATASOURCE_URL='${{ secrets.DATASOURCE_URL }}'`
is set via `.identity`

we need to add a new one secret
```
  env_secrets = {
...
    "DATASOURCE_URL": data.azurerm_key_vault_secret.key_vault_datasource_url.value,
  }

```
something like `FLYWAY_DATASOURCE_URL`
that use dns, `5432` and new param `lock_timeout=30000` that tou hare rich out